packages/base/field-component.gts

```diff
@@ -218,6 +218,10 @@ export function getBoxComponent(
       </DefaultFormatConsumer>
     </CardContextConsumer>
     <style>
+      .field-component-card.isolated-format {
+        height: 100%;
+      }
```

Comment:
I used this to make the card take up its whole container, which allowed me to use % height on the map element instead of a fixed value. It seems like this is not an ideal approach, but what’s better?

Reply:
Actually I think this might make sense. No reason why the isolated card shouldn't take up its entire container height. We can give this a try.